digiwf-integrations/digiwf-s3-integration/docs/quickstart.md

```diff
@@ -0,0 +1,46 @@
+# Quickstart
```

Comment:
Also move the relevant parts of the quickstart instructions to `docs/digiwf-integrations`